netmaster/master/network.go

```diff
@@ -499,9 +526,5 @@ func networkReleaseAddress(nwCfg *mastercfg.CfgNetworkState, ipAddress string) e
 }
 
 func hasActiveEndpoints(nwCfg *mastercfg.CfgNetworkState) bool {
-	// Uncomment the below after https://github.com/contiv/netplugin/pull/269 is merged
-	// We spin a dns container if IsDNSEnabled() == true
-	// We need to exlude that from Active EPs check.
-	//return (IsDNSEnabled() && nwCfg.EpCount > 1) || ((!IsDNSEnabled()) && nwCfg.EpCount > 0)
-	return nwCfg.EpCount > 1
+	return nwCfg.EpCount > 0
```

Comment:
@DivyaVavili might need to checked based on my earlier comment. Good to go back to the commented code.